lib/class/subsonic_api.class.php

```diff
@@ -1800,11 +1800,6 @@ public static function scrobble($input)
             $media = new $type($aid);
             $media->format();
 
-            // internal scrobbling (user_activity and object_count tables)
-            if (($submission === 'true' || $submission === '1') && $counter == 0) {
-                $media->set_played($user->id, $input['c'], array(), time());
```

Comment:
Because the way this is now updated has nothing to do with the subsonic reporting that it is played anymore, hence there is no need to limit anything. In short, for the counter in ampache (not for last.fm, this is done elsewhere), we ignore all incoming info regarding this. We instead deduce this info by reacting to changes in currently_played. As this info does not come in bulks or anything, no counter is needed.